offsets-tracker/downloader/go_wrapper.go

```diff
@@ -0,0 +1,51 @@
+package downloader
+
+import (
+	_ "embed"
+	"fmt"
+	"github.com/keyval-dev/offsets-tracker/utils"
+	"io/fs"
+	"io/ioutil"
+	"path"
+)
+
+const appName = "testApp"
```

Comment:
this conflicts with the [module name](https://github.com/open-telemetry/opentelemetry-go-instrumentation/pull/24/files#diff-1be04fea3040593d2f2a359560ac8dea215e83f826aaee271db55f61b0c026b5R1) (`testapp`) and I got this error running it:
```
2023/02/07 18:04:58 error while fetching offsets: open /tmp/testApp1757356936/testApp: no such file or directory
exit status 1

$ ls /tmp/testApp1757356936/
go.mod   go.sum   main.go  testapp
```

Reply:
I am getting the same error.